### PR TITLE
Added Reagent Grenades

### PR DESCRIPTION
**Pull request by jorgun**
## About the PR

Adds Lube and Glue Grenades. Based of Cleanades
a bundle box for a common riff raff, and and disconted for the true hero's of the station the clown and mime 

## Why / Balance

prevent sec from following you. cut of area , slow down pursuers , mute your target and prevent it from escaping.

## Technical details

smaller then the cleanade, about 1/4  so 1 cleanade can clean up 2 reagent granades.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added fun!

### CODE REVIEW

**Comment by Aidenkrz** [comment]
grenades not granades, also the commit from your assault borg pr is mixed in here.

**Comment by jorgun** [comment]
> grenades not granades, also the commit from your assault borg pr is mixed in here.
yeah i commited it to the wrong branch , thats why the other branch had an errror

**Comment by TooSillyFennec** [comment]
Not a maintainer here but, figured I'd mention this stuff anyways- not really enough for me to review here so comment it is! 

Grenades, not Granades.  Also, somewhat, not somwat. A lot of spelling errors.

The grenades aren't really priced well. Clown's Lube grenades are way too cheap compared to the clustersoap, and the box is definitely not worth 40 TC. A freedom implanter costs less by the way, making this "Fuck with the janitor" for the same price tier of an actually useful item.

Also, do remember .ftl and the locale folder exist. Every uplink item uses them, even goob unique items.

To mention the cluster soap- Just buff it instead of adding new items that won't get used. 1 decent item's better than 3 unused items that are straight up useless.

File: Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -10,6 +10,16 @@
   categories:
     -  UplinkDisruption
 
+- type: listing
+  id: UplinkBoxReagentGrenades
+  name: Box of Reagent Grenades
+  description: Contains 2 Glue granades and 2 Lube Grenades
```
Comment (a-person5660) [inline Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml:16]: "Granades" should be "grenades"

File: Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml [hunk 2]
```diff
@@ -49,3 +59,18 @@
   - !type:BuyerJobCondition
     whitelist:
     - Clown
+
+- type: listing
+  id: UplinkLubeGrenade
+  name: Lube grenades
+  description: Discounted Granades.30% off  Make the floor somwat Slippery!
```
Comment (a-person5660) [inline Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml:66]: "Granades" should be "grenades", specifying that this item is 30% off seems redundant but if you want to keep it there are more spelling errors to fix.

File: Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml [hunk 3]
```diff
@@ -49,3 +59,18 @@
...
+  productEntity: LubeGrenade
+  cost:
+    Telecrystal: 7
+  categories:
+    -  UplinkJob
+  conditions:
+  - !type:BuyerJobCondition
+    whitelist:
+    - Clown # im sure some Clown mains will spend 100 tc just to buy 14 granades
```
Comment (a-person5660) [inline Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml:75]: Another "granades", should be "grenades"

File: Resources/Prototypes/Goobstation/Entities/Weapons/Guns/ReagentGranades.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    solution:
+      reagents:
+      - ReagentId: SpaceGlue
+        Quantity: 20
+
+- type: entity
+  name: Syndicate Box of Reagent Grenades
+  parent: BoxCardboard
+  id: BoxReagentGrenades
+  description: A box full of Lube grenades .
```
Comment (a-person5660) [inline Resources/Prototypes/Goobstation/Entities/Weapons/Guns/ReagentGranades.yml:39]: Space between grenades and the period, there shouldn't be any.

File: Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml [hunk 5]
```diff
@@ -49,3 +59,18 @@
...
+    - Mime
```
Comment (a-person5660) [inline Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml:76]: You should give the mime glue grenades instead of lube grenades

**Review by a-person5660 — CHANGES_REQUESTED** [review]
There's some spelling issues and you should probably give mimes glue grenades instead of lube grenades too. I think the pricing is fine, though I don't have final say on that.

**Comment by a-person5660** [comment]
And you might wanna throw a buff for cluster soap here or make another PR for that.

File: Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml (hunk 5 above)
Comment (Piras314) [reply Resources/Prototypes/Goobstation/Catalog/uplink_catalog.yml:91]: I think both having access to both is fine

**Review by Piras314 — APPROVED** [review]

File: Resources/Prototypes/Goobstation/Entities/Weapons/Guns/ReagentGranades.yml
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Entities/Weapons/Guns/ReagentGranades.yml:1]: Change the name of this file to reagent_grenades.yml and then this is approved

**Review by Piras314 — CHANGES_REQUESTED** [review]

**Comment by Piras314** [comment]
Prices also need tweaking.

**Comment by a-person5660** [comment]
> Prices also need tweaking.

What prices need tweaking and what to?

**Comment by jorgun** [comment]
lube is now 15, the box is still 40 to prevent over use.  glue can stay at 7 as it is not gonna be as problematic.

**Review by Piras314 — CHANGES_REQUESTED** [review]
reagent_grenades not Reagent_Grenades

**Pull request merged**